testing.Makefile

```diff
@@ -76,10 +72,44 @@ testing/marketplace/deployer/envsubst: \
 		.testing/marketplace/deployer/envsubst/standard
 
 
-.PHONY: testing/all
-testing/all: \
+.PHONY: testing/integration
+testing/integration: \
 		testing/marketplace/deployer/envsubst \
 		testing/marketplace/deployer/helm_tiller_onbuild
 
 
+
+PYTHON_TEST_DIRS = $(shell find . \
+                     -path ./vendor -prune -o \
+                     -name "*_test.py" -type f -print \
+                     | xargs -n 1 dirname | sort | uniq)
+
+# Append .__pytest__ to all python test directories to generate targets.
+PYTHON_TEST_TARGETS = $(foreach f,$(PYTHON_TEST_DIRS),$(f).__pytest__)
+
+
+.PHONY: test/py
```

Comment:
should be `testing/py` to match the following target

Reply:
Good eye. :)